helpers/tistream.py

```diff
@@ -0,0 +1,21 @@
+from StringIO import StringIO
+
+class TiedStream(StringIO):
```

Comment:
IIUC this class is required to remember the original request without leading empty lines. Is there a simpler way to do this? Like copy the request before removing the empty lines?

Reply:
@krizhanovsky Probably. I'll think on it.
To be fair, empty lines are nor removed, thay are ignored.
To be fair, I can not copy request before parsing, because I do not know where it ends.

Reply:
OK, I also don't have a better idea. Could you please just leave a good comment for the class what it does and why and when we need it.